Incorrect package json exports in @equinor/fusion-framework-react-app

**Describe the bug**
The package json exports are incorrect in @equinor/fusion-framework-react-app. 
![image](https://github.com/equinor/fusion-framework/assets/89254170/961de05c-ce67-42e6-8b32-df068b5c7d52)


## My tsconfig
```JSON
{
...
    "target": "ESNext",
    "module": "ESNext",
...
}
```


Stacktrace 
```
@cc-reports/powerbi:build: src/hooks/usePbiHelpers.ts(2,29): error TS7016: Could not find a declaration file for module '@equinor/fusion-framework-react-app/http'. 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/.pnpm/@equinor+fusion-framework-react-app@5.0.0_@equinor+fusion-framework-module-http@5.2.0_@equino_qkxy5ezcdprgdphhqxzdf2hkgy/node_modules/@equinor/fusion-framework-react-app/dist/esm/http/index.js' implicitly has an 'any' type.
@cc-reports/powerbi:build:   There are types at 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/@equinor/fusion-framework-react-app/dist/types/http/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@equinor/fusion-framework-react-app' library may need to update its package.json or typings.
@cc-reports/powerbi:build: src/hooks/useWorkspaceBookmark.ts(6,8): error TS7016: Could not find a declaration file for module '@equinor/fusion-framework-react-app/bookmark'. 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/.pnpm/@equinor+fusion-framework-react-app@5.0.0_@equinor+fusion-framework-module-http@5.2.0_@equino_qkxy5ezcdprgdphhqxzdf2hkgy/node_modules/@equinor/fusion-framework-react-app/dist/esm/bookmark/index.js' implicitly has an 'any' type.
@cc-reports/powerbi:build:   There are types at 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/@equinor/fusion-framework-react-app/dist/types/bookmark/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@equinor/fusion-framework-react-app' library may need to update its package.json or typings.
@cc-reports/powerbi:build: src/fusion-framework/frameworkConfig.ts(1,34): error TS7016: Could not find a declaration file for module '@equinor/fusion-framework-react-app'. 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/.pnpm/@equinor+fusion-framework-react-app@5.0.0_@equinor+fusion-framework-module-http@5.2.0_@equino_qkxy5ezcdprgdphhqxzdf2hkgy/node_modules/@equinor/fusion-framework-react-app/dist/esm/index.js' implicitly has an 'any' type.
@cc-reports/powerbi:build:   There are types at 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/@equinor/fusion-framework-react-app/dist/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@equinor/fusion-framework-react-app' library may need to update its package.json or typings.
@cc-reports/powerbi:build: src/fusion-framework/createRender.tsx(5,8): error TS7016: Could not find a declaration file for module '@equinor/fusion-framework-react-app'. 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/.pnpm/@equinor+fusion-framework-react-app@5.0.0_@equinor+fusion-framework-module-http@5.2.0_@equino_qkxy5ezcdprgdphhqxzdf2hkgy/node_modules/@equinor/fusion-framework-react-app/dist/esm/index.js' implicitly has an 'any' type.
@cc-reports/powerbi:build:   There are types at 'C:/Users/gustav.eikaas/repo/cc-reports/node_modules/@equinor/fusion-framework-react-app/dist/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@equinor/fusion-framework-react-app' library may need to update its package.json or typings.
```

Here is an example of how it should be done:
https://github.com/equinor/fusion-framework/blob/main/packages/framework/package.json#L6


## Tool for checking if everything is configured correctly
https://arethetypeswrong.github.io/?p=%40equinor%2Ffusion-framework-react-app%405.0.0

--- a/.changeset/tidy-socks-clean.md
+++ b/.changeset/tidy-socks-clean.md
@@ -1,0 +1,5 @@
+---
+"@equinor/fusion-framework-react-app": patch
+---
+
+Fixes type exports in package

--- a/packages/react/app/package.json
+++ b/packages/react/app/package.json
@@ -1,46 +1,45 @@
 {
     "name": "@equinor/fusion-framework-react-app",
-    "version": "5.0.0",
+    "version": "5.0.2",
     "description": "",
-    "main": "dist/esm/index.js",
-    "exports": {
-        ".": "./dist/esm/index.js",
-        "./bookmark": "./dist/esm/bookmark/index.js",
-        "./context": "./dist/esm/context/index.js",
-        "./feature-flag": "./dist/esm/feature-flag/index.js",
-        "./framework": "./dist/esm/framework/index.js",
-        "./http": "./dist/esm/http/index.js",
-        "./msal": "./dist/esm/msal/index.js",
-        "./navigation": "./dist/esm/navigation/index.js",
-        "./widget": "./dist/esm/widget/index.js"
-    },
+    "main": "./dist/esm/index.js",
     "types": "./dist/types/index.d.ts",
-    "typesVersions": {
-        "*": {
-            "bookmark": [
-                "dist/types/bookmark/index.d.ts"
-            ],
-            "context": [
-                "dist/types/context/index.d.ts"
-            ],
-            "feature-flag": [
-                "dist/types/feature-flag/index.d.ts"
-            ],
-            "framework": [
-                "dist/types/framework/index.d.ts"
-            ],
-            "http": [
-                "dist/types/http/index.d.ts"
-            ],
-            "msal": [
-                "dist/types/msal/index.d.ts"
-            ],
-            "navigation": [
-                "dist/types/navigation/index.d.ts"
-            ],
-            "widget": [
-                "dist/types/widget/index.d.ts"
-            ]
+    "exports": {
+        ".": {
+            "types": "./dist/types/index.d.ts",
+            "import": "./dist/esm/index.js"
+        },
+        "./bookmark": {
+            "types": "./dist/types/bookmark/index.d.ts",
+            "import": "./dist/esm/bookmark/index.js"
+        },
+        "./context": {
+            "types": "./dist/types/context/index.d.ts",
+            "import": "./dist/esm/context/index.js"
+        },
+        "./feature-flag": {
+            "types": "./dist/types/feature-flag/index.d.ts",
+            "import": "./dist/esm/feature-flag/index.js"
+        },
+        "./framework": {
+            "types": "./dist/types/framework/index.d.ts",
+            "import": "./dist/esm/framework/index.js"
+        },
+        "./http": {
+            "types": "./dist/types/http/index.d.ts",
+            "import": "./dist/esm/http/index.js"
+        },
+        "./msal": {
+            "types": "./dist/types/msal/index.d.ts",
+            "import": "./dist/esm/msal/index.js"
+        },
+        "./navigation": {
+            "types": "./dist/types/navigation/index.d.ts",
+            "import": "./dist/esm/navigation/index.js"
+        },
+        "./widget": {
+            "types": "./dist/types/widget/index.d.ts",
+            "import": "./dist/esm/widget/index.js"
         }
     },
     "scripts": {

